pkg/providers/vsphere/virtualmachine/backup.go

```diff
@@ -46,6 +47,12 @@ type PVCDiskData struct {
 	AccessModes []corev1.PersistentVolumeAccessMode
 }
 
+// ClassicDiskData contains the data of a classic (static) disk attached to VM.
+type ClassicDiskData struct {
```

Comment:
I think we talked about moving this, and the PVCData to the types, so the restore workflow can use it?

Also, move all the backup/restore related fields to a new files in the api/ repo if you want.

Reply:
Thanks! Updated as suggested in the latest commit.